src/dfmt/formatter.d

```diff
@@ -471,15 +474,18 @@ private:
         import dfmt.editorconfig : OptionalBoolean;
 
         immutable bool isCase = astInformation.caseEndLocations.canFindIndex(current.index);
-        immutable bool isAttribute = astInformation.attributeDeclarationLines.canFindIndex(current.line);
+        immutable bool isAttribute = astInformation.attributeDeclarationLines.canFindIndex(
```

Comment:
Split off into #181  I have a commit, which splits at "=", but it breaks other tests at the moment.